guide/de/fdl-appendix.xml

```diff
@@ -1,566 +1,467 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE appendix SYSTEM "gnc-docbookx.dtd">
-<!--
-     The GNU Free Documentation License 1.1 in DocBook
-     Markup by Eric Baudais <baudais@okstate.edu>
-     Maintained by the GNOME Documentation Project
-     http://developer.gnome.org/projects/gdp
-     Version: 1.0.1
-     Last Modified: Nov 16, 2000
--->
+<!DOCTYPE appendix PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
 <appendix id="fdl">
-  <appendixinfo>
-    <releaseinfo>
-      Version 1.1, March 2000
-    </releaseinfo>
-
-    <copyright>
-      <year>2000</year><holder>Free Software Foundation, Inc.</holder>
-    </copyright>
-
-    <legalnotice id="fdl-legalnotice">
-      <para><address>Free Software Foundation, Inc. <street>59 Temple Place,
-        Suite 330</street>, <city>Boston</city>, <state>MA</state>
-        <postcode>02111-1307</postcode>  <country>USA</country></address>
-        Everyone is permitted to copy and distribute verbatim copies of this license document, but
-        changing it is not allowed.
-      </para>
-    </legalnotice>
-  </appendixinfo>
-
   <title>GNU Free Documentation License</title>
```

Comment:
I can't find a German translation of FDL in maint. 
Do you mean I should copy and add it?

Reply:
The source package is gnome-desktop-40.4-1.1. On opensuse that is split in several parts and you need gnome-desktop-lang.

Because the conventions of the package follow https://www.freedesktop.org/wiki/Specifications/help-spec/, we should consider to fix https://bugs.gnucash.org/show_bug.cgi?id=798166 before.